operator/src/main/resources/application.properties

```diff
@@ -38,6 +38,14 @@ ingresscontroller.peak-throughput-percentage=40
 # old sizing - this should go higher, but we should need more memory
 ingresscontroller.max-ingress-connections=75000
 
+# Enables haproxy option contstats in the kas ingress so that stats are reported live rather than at connection close (RFE-3007 provide a proper mechanism to enable this option).
+ingresscontroller.ingress-container-command=/usr/bin/bash,-c,awk '{print $0} /^defaults$/ {print \"  option contstats\"}' < $TEMPLATE_FILE > /tmp/haproxy-config.template; exec /usr/bin/openshift-router --v=2 --template /tmp/haproxy-config.template
+# Disconnect established connections after a haproxy reconfiguration (NE-879 should eliminate the need for this)
+ingresscontroller.hard-stop-after=5s
+# Coalesce up-to reloadInterval worth of haproxy updates.  This prevents a flurry of haproxy restarts (1 per OpenShift Route) as each kafka
+# instance is created (NE-879 should eliminate the need for this).
+ingresscontroller.reload-interval-seconds=60
```

Comment:
Was 60s selected based on anything in particular? 

I was going to suggest 30s but thinking a bit further it might be wise to make sure the duration is greater than the period between prometheus scrapes.

I'm just conscious that it will slow down the availability of instance changes. I don't have a good view on how that would play with things like the console/cli saying an instance is available.

Reply:
The operator creates the openshift routes sequentially, so my aim was to ensure that those routes (n [one for each broker] + 1 bootstrap + 1 admin) have a reasonable chance of being coalesced into a single haproxy restart.  Experiment 30s seemed to be enough for a 3 broker instance.  I didn't test 6.  I doubled it for a safety margin.

During provisioning IIRC Route53 registration will go on in parallel.  In practice, that takes longer than 1 minute, so I don't expect this to impact provision time.  Unfortunately, we can't test this part in development.    Route 53 isn't part of the solution until Stage.

Reply:
Cool that seems reasonable for prod,I don't suppose it makes any real difference in practice for kas-installer environments either.